src/Bridge.ts

```diff
@@ -115,6 +121,24 @@ export class Bridge {
         await this.tokenStore.load();
         const connManager = this.connectionManager = new ConnectionManager(this.as,
             this.config, this.tokenStore, this.commentProcessor, this.messageClient, this.github);
+    
+        if (this.config.provisioning) {
+            const routers = [];
+            if (this.config.jira) {
+                routers.push({
+                    route: "/v1/jira",
+                    router: new JiraProvisionerRouter(this.config.jira, this.tokenStore).getRouter(),
+                });
+                this.connectionManager.registerProvisioningConnection(JiraProjectConnection);
+            }
+            if (this.config.github) {
+                routers.push({
+                    route: "/v1/github",
+                    router: new GitHubProvisionerRouter(this.config.github, this.tokenStore).getRouter(),
+                });
+            }
```

Comment:
looks like this is missing a registration of some kind. I also had to fix the provisioner details of the GithubRepo connection:

```diff
diff --git a/src/Bridge.ts b/src/Bridge.ts
index 2db2d28..ca1e057 100644
--- a/src/Bridge.ts
+++ b/src/Bridge.ts
@@ -136,6 +136,7 @@ export class Bridge {
                     route: "/v1/github",
                     router: new GitHubProvisionerRouter(this.config.github, this.tokenStore).getRouter(),
                 });
+                this.connectionManager.registerProvisioningConnection(GitHubRepoConnection);
             }
             this.provisioningApi = new Provisioner(this.config.provisioning, this.connectionManager, this.as.botIntent, routers);
         }
diff --git a/src/Connections/GithubRepo.ts b/src/Connections/GithubRepo.ts
index 21e7075..7025d5a 100644
--- a/src/Connections/GithubRepo.ts
+++ b/src/Connections/GithubRepo.ts
@@ -682,7 +682,7 @@ ${event.release.body}`;
         return {
             service: "github",
             eventType: GitHubRepoConnection.CanonicalEventType,
-            type: "JiraProject",
+            type: "GithubRepo",
             // TODO: Add ability to configure the bot per connnection type.
             botUserId: botUserId,
         }
```